tests/regression/tests/REQUEST-942-APPLICATION-ATTACK-SQLI/942440.yaml

```diff
@@ -319,7 +319,7 @@ tests:
             method: "GET"
             port: 80
             version: "HTTP/1.1"
-            uri: "/get/?fbclid=IwAR1dug0BYxe0ukhZ2vKrdQwLAxVFRJ--Q2Y7OBJE_0uId9-Eh-sJWLdVk2E"
+            uri: "/get?fbclid=IwAR1dug0BYxe0ukhZ2vKrdQwLAxVFRJ--Q2Y7OBJE_0uId9-Eh-sJWLdVk2E"
```

Comment:
Why?

Reply:
Because most of the tests are using uri `/get?..` instead of `/get/?..`.

Reply:
Is this explanation sufficient for you?

Reply:
Yes.